src/Http/Response.php

```diff
@@ -2233,6 +2233,21 @@ public function withFile($path, array $options = [])
     }
 
     /**
+     * Convenience method to set a string into the response body
+     *
+     * @param string $string The string to be sent
+     * @return static
+     */
+    public function withStringBody($string)
+    {
+        $new = clone $this;
+        $new->_createStream();
```

Comment:
`_createStream()` is a protected method. PHP still allows it to be used just because you are in instance of same class?